policy/resolver.go

```diff
@@ -1281,8 +1305,6 @@ func (s *LocalServices) jobsToQueries(ctx context.Context, policyMrn string, cac
 			rj := reportingjobs[i]
 			// (2) Scoring Queries handling
 			if !isDataQuery {
-				rj.QrId = codeID
```

Comment:
we now assign the code id directly as a QrId in `addCheckJob` so this is no longer needed